tools/ocean/argo_decoder.xml

```diff
@@ -0,0 +1,63 @@
+<tool id="coriolis_data_processing_chain_for_argo_floats" name="coriolis data processing" version="@VERSION@">
+    <description>chain for argo floats</description>
+    <macros>
+        <token name="@VERSION@">061a</token>
+        <token name="@VERSION_DOCKER@">@VERSION@-galaxy</token>
+    </macros>
+    <requirements>
+        <container type="docker">ghcr.io/euroargodev/coriolis-data-processing-chain-for-argo-floats:@VERSION_DOCKER@</container>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        
+        ## prepare outputs directory
+        mkdir -p /mnt/data/output/log &&
+        chmod -R 770 /mnt/data/output &&
+
+        ## load inputs into data directory
+        cp '$raw_data' '/tmp/raw_data.zip' &&
+        unzip /tmp/raw_data.zip -d /mnt/data &&
+
+        ## execute decoder script
+        ##$APP_HOME/$APP_RUN_FILE /mnt/runtime 'rsynclog' 'all' 'configfile' '/app/config/_argo_decoder_conf_ir_sbd.json' 'xmlreport' 'argo-decoder-report.xml' 'floatwmo' '$float_wmo' 'PROCESS_REMAINING_BUFFERS' '1' &&
+        /app/run_decode_argo_2_nc_rt.sh /mnt/runtime 'rsynclog' 'all' 'configfile' '/app/config/_argo_decoder_conf_ir_sbd.json' 'xmlreport' 'argo-decoder-report.xml' 'floatwmo' '$float_wmo' 'PROCESS_REMAINING_BUFFERS' '1' &&
+
+        ## copy (because we need to be in working directory) and create output archive
+        cp -R /mnt/data/output/* . &&
```

Comment:
You could zip it in the /mnt dir and just copy the zip file to $outputs_data

Reply:
`cp argo-decoder-outputs.zip '$outputs_data'` will work.

Or you can use `pushd` and `popd`

Reply:
ok we'll use '$outputs_data', thanks.

Reply:
@Marie59 can you replace next code ?
```bash
        cp -R /mnt/data/output/* . &&
        zip -r argo-decoder-outputs.zip .
```
by :
```bash
        cd /mnt/data/output &&
        zip -r argo-decoder-outputs.zip . &&
        cp argo-decoder-outputs.zip '$outputs_data'
``